contrib/containers/deploy/Dockerfile

```diff
@@ -5,11 +5,11 @@ LABEL description="Dockerised DashCore, built from CI"
 ARG USER_ID
 ARG GROUP_ID
 
-ENV HOME=/home/dash
+ENV HOME="/home/dash"
```

Comment:
this particular case no really need quotes :D

Reply:
well you told me to add quotes!

Reply:
do it wisely!